ui/src/app/job-details/tabs/tabs.component.html

```diff
@@ -128,13 +128,9 @@
     <mat-tab *ngIf="hasOutputs()" label="Outputs">
       <jm-resources-table [entries]="job.outputs"></jm-resources-table>
     </mat-tab>
-    <!-- Disable the graph view tab so the indicator doesn't switch on click -->
-    <mat-tab *ngIf="hasTimingUrl()" label="Graph View" [disabled]=true>
-      <ng-template mat-tab-label>
-        <a style="all: unset" href="{{ job.extensions.timingUrl }}" target="_blank">
```

Comment:
Can we now remove this extension from the Cromwell shim code?

Reply:
Good point – removed.